web/modules/custom/joinup_front_page/joinup_front_page.module

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations for the Joinup front page module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function joinup_front_page_form_menu_edit_form_alter(&$form, FormStateInterface $form_state, $form_id): void {
+  /** @var \Drupal\system\Entity\Menu $menu */
+  $menu = $form_state->getFormObject()->getEntity();
+  if ($menu->id() !== 'front-page') {
+    return;
+  }
+
+  $form['#title'] = t('Front page pinned entities');
```

Comment:
Let's not mention "entities" in user facing text. How about:

```
Pinned content
```

Reply:
The entities I used was because we might have rdf_entities as well and `content` in the Drupal world refers to nodes :/

Reply:
`Pinned items`????